### ISSUE TITLE
`meanwile` need to be `meanwhile` in proxychains.conf

### ISSUE THREAD

**Issue by yeshiqing** [rofl0r/proxychains-ng#474]
https://github.com/rofl0r/proxychains-ng/blob/060801d8c8613692469c8d1ccfc2598fb8a2992a/src/proxychains.conf#L159

**Comment by rofl0r** [comment]
yes, people make typoes, but i dont want to clutter the commit history with typo fix spam. maybe i'll fix all of them at once someday.
